### PR TITLE
[ENHANCEMENT] Added sort-by functionality to getUsersBySystem in UserRepository

### PR DESCRIPTION
**Pull request by turbo-ele**
**informations:**

| q | a |
| --- | --- |
| Fixed tickets | none |
| Related PRs | none |
| BC breaks | none |
| Documentation PR | none |

### CODE REVIEW

File: src/Sulu/Bundle/ContactBundle/Controller/ContactController.php [hunk 1]
```diff
@@ -751,8 +752,9 @@ protected function getContactManager()
      */
     protected function getContactsByUserSystem()
     {
+        /** @var UserRepository $repo */
         $repo = $this->get('sulu_security.user_repository');
-        $users = $repo->getUserInSystem();
+        $users = $repo->getUserInSystem(['contact.firstName' => 'ASC', 'contact.lastName' => 'ASC']);
```
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Controller/ContactController.php]: Wouldn't you usually like them to be ordered by last name first?

**Comment by danrot** [comment]
Tests missing.

**Comment by danrot** [comment]
@turbo-ele Do you still need this change?

**Comment by turbo-ele** [comment]
yes

**Comment by danrot** [comment]
Are you going to add the tests one day? :stuck_out_tongue:

**Comment by turbo-ele** [comment]
probably

**Comment by danrot** [comment]
@turbo-ele ping

**Comment by danrot** [comment]
Closing, since it probably won't be finished.

**Pull request closed without merging**